packages/core/src/mm/ovl_patch/kaleido_scope.c

```diff
@@ -334,15 +348,16 @@ static u32 sCustomIcons[] = {
     ITEM_MM_BOOTS_HOVER,
     ITEM_MM_TUNIC_GORON,
     ITEM_MM_TUNIC_ZORA,
+    ITEM_MM_HAMMER,
 };
 
-s8 gPlayerFormCustomItemRestrictions[5][8] =
+s8 gPlayerFormCustomItemRestrictions[5][ITEM_MM_CUSTOM_MAX - ITEM_MM_CUSTOM_MIN] =
 {
     { 0, 0, 0, 0, 0, 0, 0, 0 },
     { 0, 0, 0, 0, 0, 0, 0, 0 },
     { 0, 0, 0, 0, 0, 0, 0, 0 },
     { 0, 0, 0, 0, 0, 0, 0, 0 },
-    { 1, 1, 1, -1, -1, -1, -1, 0 },
+    { 1, 1, 1, -1, -1, -1, -1, -1 },
```

Comment:
`-1` means it'll be forced to be enabled (such as in situations like being underwater or indoors), that's why it's used for boots and tunics. If you set it to `1` it should match the behavior of other items.